src/main/java/cz/muni/fi/pa165/entity/Ticket.java

```diff
@@ -0,0 +1,72 @@
+package cz.muni.fi.pa165.entity;
+
+import lombok.*;
+
+import javax.persistence.*;
+import javax.validation.constraints.*;
+import java.time.LocalDate;
+import java.util.Objects;
+
+/**
+ * Ticket to a specific performance
+ * @author Tomas Rudolf
+ */
+@Entity
+@Table(name = Ticket.TABLE_NAME)
+public class Ticket {
```

Comment:
Kde má nějakej kód, kterým se bude prokazovat divák při vstupu? Nebo bude používat ID? To se mi zdá, že by lidi zjistili dost brzo a oblbli to.

Reply:
ID jako carovy kod mi neprijde jako neco, co by nekdo chtel oblbnout. Nebo ty jsi nekde videl, ze carovy kod nestaci?

Reply:
Ale jo, jen se bojím, že by nějací ajťáci ten kód přečetli a mohlo by jim to přijít divný/podezřelý a pokud by těch představení bylo málo, mohly by bejt schopní vyrobit vstupenku bez toho, aby si ji zaplatili - použijou id tý svý + 1 a jelikož vstupenky vznikaj až při zaplacení, pokud trefjej id, bude už určitě zaplacená, dost možná dokonce i na správný představení. Jasně, náhodnýmu milovníku divadla to může bejt asi jedno a na tohle nepřijde, ale zas tak velká změna to není, aby to jako nemohlo bejt řešený už teď, skoro bych se vsadil, že jako na potvoru u review taky bude nějakej takovej paranoidní člověk.

Reply:
Jo, ale tohle odsledujes jen pri review nebo kdyz koupis 3 vstupenky naraz, rovnou je zaplatis a dostanes n, n+1 a n+2. Koukal jsem jak se da delat UUID jako @Id, ale moc se mi to nezda, tak jsem spis za polozku barcode, co bude 128 bitu cislo jako nejefektivnejsi ulozeni v databazi a pro Javu to bude UUID objekt